content/en/blogs/TiDB-on-KubeSphere-using-qke.md

```diff
@@ -152,15 +154,15 @@ Now that we have our apps ready, we may need to focus more on observability. Kub
 
 ![view-tikv-loads](https://ap3.qingstor.com/kubesphere-website/docs/20201027141541.png)
 
-5. Relevant Pods are also listed. As you can see, tidb-cluster contains three pd Pods, two tidb Pods and 3 tikv Pods.
+5. Relevant Pods are also listed. As you can see, tidb-cluster contains three pd Pods, two TiDB Pods and 3 TiKV Pods.
```

Comment:
same here

Reply:
Please refer to the TiDB's technical writer:

![image](https://user-images.githubusercontent.com/40452856/98123960-89a05280-1eed-11eb-97cc-d5c345dea54e.png)